src/site/content/en/blog/use-thumbor/index.md

```diff
@@ -0,0 +1,211 @@
+</figure>
+
+
+2. Resize the image to a width of 1000 pixels, then resize the image to 10% of its current size: <a href="http://34.67.235.246:8888/unsafe/1000x/filters:proportion(.1)/https://web.dev/backdrop-filter/hero.jpg" target="_blank" rel="noreferrer">http://34.67.235.246:8888/unsafe/1000x/filters:proportion(.1)/https://web.dev/backdrop-filter/hero.jpg</a>
+
+<figure class="w-figure  w-figure--center">
+  <img src="./Use-thumbor10.jpg" alt="Image that is 100 pixels wide" class="w-screenshot">
+  <figcaption>Image resized to a width of 100 pixels</figcaption>
+</figure>
+
+
+These methods are just a few of Thumbor's many cropping and resizing options. To read about other options, check out [Usage](https://github.com/thumbor/thumbor/wiki/Usage).
+
+
+### File Formats
+
+The [format](https://thumbor.readthedocs.io/en/latest/format.html) filter converts images to `jpeg`, `webp`, `gif`, or `png`. Keep in mind that if you're optimizing for performance you should [use either JPEG or WebP](https://developers.google.com/web/fundamentals/performance/optimizing-content-efficiency/image-optimization) as PNG and GIF files tend to be significantly larger and do not compress as well.
+
+Try it now:
+
+1. Convert the image to WebP. If you open the **Network** panel of DevTools the document's **Content-Type response header** shows that the server returned a WebP image: <a href="http://34.67.235.246:8888/unsafe/filters:format(webp)/https://web.dev/backdrop-filter/hero.jpg" target="_blank" rel="noreferrer">http://34.67.235.246:8888/unsafe/filters:format(webp)/https://web.dev/backdrop-filter/hero.jpg</a>
+
+<figure class="w-figure  w-figure--center">
+  <img src="./Use-thumbor11.jpg" alt="DevTools screenshot showing the content-type (WebP) of an image" class="w-screenshot">
+  <figcaption>The `content-type` header in DevTools</figcaption>
+</figure>
+
+
+## Next Steps
+
+Try out other [filters](https://thumbor.readthedocs.io/en/latest/filters.html) and transformations on the `hero.jpg` image.
+
+If you're following along using your own Thumbor installation, check out the appendix below that explains how and why to use the `thumbor.conf` file.
+
+
+## Appendix: `thumbor.conf`
+
+Many of the configuration options discussed in this post, plus many others, can be established as defaults by setting up and using a `thumbor.conf` configuration file. Settings in the `thumbor.conf` file will be applied to all images unless overridden by the URL string parameters.
+
+1. Run the `thumbor-config` command to create a new `thumbor.conf` file.
+
+```bash
+thumbor-config > ./thumbor.conf
+```
+
+2. Open your new `thumbor.conf` file. The `thumbor-config` command generated a file that lists and explains all Thumbor configuration options.
+
+3. Configure settings by uncommenting lines and changing the default values.
+You may find it useful to set the following settings:
+* `QUALITY`
+* `AUTO_WEBP`
+* `MAX_WIDTH` and `MAX_HEIGHT`
+* `ALLOW_ANIMATED_GIFS`
```

Comment:
These bullet-list items are formatting weirdly but I believe I was the one that suggested changing it to a bulleted list so I'll also fix this in a subsequent PR

cc @robdodson

Reply:
I figured out the fix for this so need for a separate PR.